source/content/redis.md

```diff
@@ -51,7 +51,9 @@ All plans except for the Basic plan can use Redis. Sandbox site plans can enable
     ```php:title="object-cache.php"
     <?php
     # This is a Windows-friendly symlink
```

Comment:
@ccharlton while we're looking at this block, does this comment belong here? It doesn't seem to match the code.

Reply:
Confirmed by @kyletaylored